### DISCUSSION TITLE
How do I combine `cd` and fzf to search through directories only?

### DISCUSSION THREAD

**Discussion by nimser** [PatrickF1/fzf.fish#268] [Q&A] [answered]
Hi, using `cd` then CTRL+ALT+F shows a lot of irrelevant fuzzy inputs because it also display files. How do I get only directories?

**Comment by PatrickF1** [comment, marked as answer]
Pass in `--type=directory` to fd using https://github.com/PatrickF1/fzf.fish#change-the-files-searched.

**Reply by nimser** [reply]
Not a solution for me as this would mean no files show up when I use fzf with CTRL+ALT+F, with the intent of opening a file for example.
Could a better be to add a CTRL+ALT+D (D for Directories) and probably corresponding options/variables? Is there a wiki section on how to achieve this?

**Reply by PatrickF1** [reply]
fzf.fish is focused on completing commands in general, so I don't want to make a new feature just for cd.
That said, if you don't mind hacky, you could keep the current search directory function and then create your own key binding that sets `fzf_fd_opts=--type=directory` before calling the search directory function. Reading the source of https://github.com/PatrickF1/fzf.fish/blob/main/functions/fzf_configure_bindings.fish may help.
